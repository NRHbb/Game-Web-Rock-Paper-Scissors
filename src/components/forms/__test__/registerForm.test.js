import RegisterForm from '../RegisterForm'
import { screen, render, cleanup, getByLabelText } from '../../../tests/testUtils'

const registerForm = render(<RegisterForm/>)

it('renders register form', () => {
  expect(registerForm).toMatchSnapshot()
})
