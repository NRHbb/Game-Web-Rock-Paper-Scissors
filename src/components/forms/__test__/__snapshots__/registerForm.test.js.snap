// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders register form 1`] = `
<Provider
  store={
    {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
